lib/stat_tracker.rb

```diff
@@ -213,8 +189,8 @@ def games_by_team(team_id)
     end
   end
 
-  def get_opponent_id(game, team_id)
-    @games_manager.get_opponent_id(game, team_id)
+  def get_opponent_id(gameid, teamid)
```

Comment:
Can you just put an underscore between game/team and "id" here?

Reply:
added! thanks